content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -83,33 +83,32 @@ In order to build Atom from source, you need to have a number of other requireme
 
 * macOS 10.9 or later
-* Node.js 6.x or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
-* npm 3.10.x or later (run `npm install -g npm`)
-* Python v2.7.x
+* Node.js 10.12 or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
+* npm 6 or later (run `npm install -g npm`)
+* Python v2.7.x or 3.5+
 * Command Line Tools for [Xcode](https://developer.apple.com/xcode/downloads/) (run `xcode-select --install` to install)
 
 {{/mac}}
 
 {{#windows}}
 
-* Node.js 6.9.4 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
+* Node.js 10.12 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
 * Python v2.7.x
   * The `python.exe` must be available at `%SystemDrive%\Python27\python.exe`. If it is installed elsewhere create a symbolic link to the directory containing the `python.exe` using: `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`
 * 7zip (7z.exe available from the command line) - for creating distribution zip files
 * C++ build tools, either:
-  * [Visual C++ Build Tools 2015](https://visualstudio.microsoft.com/vs/older-downloads/)
-  * [Visual Studio 2013 Update 5](https://www.visualstudio.com/downloads/) (Express Edition or better)
-  * [Visual Studio 2015](https://www.visualstudio.com/downloads/) (Community Edition or better)
-  * **Unsupported** but more convenient for some who know Node: [windows-build-tools](https://www.npmjs.com/package/windows-build-tools)
+  * [Visual C++ Build Tools 2015, 2017 or 2019](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+  * [Visual Studio 2015, 2017 or 2019](https://www.visualstudio.com/downloads/) (Community Edition or better)
+  * [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) (From an elevated CMD.exe or Powershell window (right click and "run as Administrator") do: `npm install --global --production windows-build-tools` to install)
 
 
   Also ensure that:
 
   * The default installation folder is chosen so the build tools can find it
   * If using Visual Studio make sure Visual C++ support is selected/installed
-  * If using Visual C++ Build Tools make sure Windows 8 SDK is selected/installed
+  * If using Visual C++ Build Tools make sure a Windows SDK (Windows 8 SDK or Windows 10 SDK) is selected/installed
   * A `git` command is in your path
-  * Set the `GYP_MSVS_VERSION` environment variable to the Visual Studio/Build Tools version (`2013` or `2015`) e.g. ``[Environment]::SetEnvironmentVariable("GYP_MSVS_VERSION", "2015", "User")`` in PowerShell (or set it in Windows advanced system settings).
+  * Set the `GYP_MSVS_VERSION` environment variable to the Visual Studio/Build Tools version (`2015`, `2017`, or `2019`.) e.g. ``[Environment]::SetEnvironmentVariable("GYP_MSVS_VERSION", "2015", "User")`` in PowerShell (or set it in Windows advanced system settings).
```

Comment:
Setting `GYP_MSVS_VERSION` is usually unnecessary, if you've installed to the default location.

node-gyp has [gotten better at locating Visual Studio](https://github.com/nodejs/node-gyp/pull/1762) and its components.

Maybe leave this for the troubleshooting area instead. It's already mentioned there.